virttest/tests/unattended_install.py

```diff
@@ -1142,6 +1142,37 @@ def string_in_serial_log(serial_log_file_path, string):
         return False
 
 
+def attempt_to_log_useful_files(test, vm):
+    """
+    Uses "vm.serial_console" to look in the system for useful log files
+    """
+    console = vm.serial_console
+    for path_glob in ["/*.log", "/tmp/*.log", "/var/tmp/*.log"]:
+        try:
+            paths = console.cmd("ls -1 %s" % path_glob)
+        except Exception as details:
+            continue
+        for path in paths.splitlines():
+            if not path:
+                continue
+            if path.startswith(os.path.sep):
+                rel_path = path[1:]
+            else:
+                rel_path = path
+            dst = os.path.join(test.outputdir, vm.name, rel_path)
```

Comment:
is it `test.debugdir` ?

Reply:
If you mean `test.outputdir()` is `results/job-2018*/test-results/1-*unattended_install*/` which is the right place.